crypto/cmp/cmp_ctx.c

```diff
@@ -919,12 +919,12 @@ int OSSL_CMP_CTX_set_option(OSSL_CMP_CTX *ctx, int opt, int val)
         ctx->keep_alive = val;
         break;
     case OSSL_CMP_OPT_MSG_TIMEOUT:
-        if (val < 0)
+        if (val <= 0)
             val = 0;
         ctx->msg_timeout = val;
         break;
     case OSSL_CMP_OPT_TOTAL_TIMEOUT:
-        if (val < 0)
+        if (val <= 0)
             val = 0;
```

Comment:
Thank you for taking care of this Coverity issue.

When looking at the code at the beginning of this function, it turns out that the case `val < 0` is already covered for both timeout settings by
```
    default:
        min_val = 0;
        break;
    }
    if (val < min_val) {
        ERR_raise(ERR_LIB_CMP, CMP_R_VALUE_TOO_SMALL);
        return 0;
    }
```
so the complaint by Coverity about dead code is actually right (thus not a false positive).

Please just remove the two occurrences of
```
if (val < 0)
    val = 0;
```

Reply:
I figured you added the checks for clarity & sanity.  Happy to remove them.